libs/feature-workspaces/src/lib/projects/projects.component.ts

```diff
@@ -48,6 +57,21 @@ export class ProjectsComponent implements OnInit {
         return { ...w, projects };
       })
     );
+
+    if (this.settings.showDocs) {
+      this.docsSubscription = this.route.params
+        .pipe(switchMap(p => this.workspaceDocsGQL.fetch({ path: p.path })))
+        .subscribe(r => {
```

Comment:
We should set this as a observable as a class member and subscribe within the template